content/docs/user-guide/assets/pipeline/asset-builders.md

```diff
@@ -1,71 +1,190 @@
-| input              | For non-resident mode, full path to the file containing the serialized job request.                                                                                                                           |
-| output             | For non-resident mode, full path to the file to write the job response to.                                                                                                                                    |
-| debug              | Debug mode for the create and process job of the specified file. <br/>Debug mode optionally uses `-input`, `-output`, `-module`, `-port` and `-gameroot`. For example: `-debug Objects\Tutorials\shapes.fbx`. |                                                                                                                                  
-| debug_create       | Debug mode for the create job of the specified file.                                                                                                                                                          |
-| debug_process      | Debug mode for the process job of the specified file.                                                                                                                                                         |
-| tags               | Additional tags to add to the debug platform for job processing. One tag can be supplied per option.                                                                                                          |
-| platform           | Platform to use for debugging, for example, `pc` |
-
+---
+linkTitle: Asset Builders 
+title: Asset Builders
+description: Asset Builders use configurable rules to generate product assets from jobs that are dispatched by Asset Processor in Open 3D Engine (O3DE).
+weight: 400
+toc: true
+---
+
+An **Asset Builder** is a bundle of code that **Asset Processor** runs to generate product assets. Asset Builders can be distributed as Gems and might use third-party libraries to process assets. The **Scene Processing** Gem, for example, contains the Asset Builders that process `.fbx` files, and uses the [Open Asset Import Library](https://github.com/assimp/assimp) to parse `.fbx` files.
+
+{{< note >}}
+The Open Asset Import Library supports [many scene formats](https://github.com/assimp/assimp/blob/master/doc/Fileformats.md). You can experiment with additional formats by editing `o3de/Registry/sceneassetimporter.setreg` and adding format extensions to the `"SupportedFileTypeExtensions"` list.
+{{< /note >}}
+
+## Anatomy of an asset builder
+
+Asset Builders have three core components: a **Descriptor** for the builder, and handlers for **Create Jobs** and **Process Job** requests.
+
+![Simple Job](/images/user-guide/assets/pipeline/asset_builders/asset_builder_basics.png)

```

Comment:
We can embed SVG versions of these files. If you replace the PNGs with SVGs, I can create an issue to revisit this doc and make some adjustments so they scale nicely.

Reply:
I don't have SVG versions of those images